API/DTOs/ProgressDto.cs

```diff
@@ -21,7 +21,7 @@ public class ProgressDto
     /// </summary>
     public string BookScrollId { get; set; }
     /// <summary>
-    /// Last time the progress was synced from UI or external app
+    /// Last time in Utc the progress was synced from UI or external app
```

Comment:
This comment doesn't make sense for within Kavita. It would be more correct to say, Last time Chapter was read in Utc.

Reply:
ok